daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -445,6 +445,21 @@ damlStartTests getDamlStart =
                           ]
                   contents <- readFileUTF8 (projDir </> "output.json")
                   lines contents @?= ["{", "  \"_1\": 0,", "  \"_2\": 1", "}"]
+        , testCase "daml export script" $ do
+              DamlStartResource {projDir} <- getDamlStart
+              withTempDir $ \exportDir -> do
+                  withCurrentDirectory projDir $ do
+                      callCommand $ unwords
+                          [ "daml ledger export"
+                          , "--party Alice"
```

Comment:
Does moving `--party` after `script` work as well? I expect for users it’s probably more intuitive in most cases to throw everything at the end.

Reply:
That doesn't work by default. There's an optparse-applicative option ([`subparserInline`](https://hackage.haskell.org/package/optparse-applicative-0.16.1.0/docs/Options-Applicative.html#v:subparserInline)) to enable this. It's a global switch though. At a quick test it seems to cause issues with other commands. E.g. it breaks `daml start` with 
```
Invalid option `--dar'
Usage: daml run-jar JAR [--logback-config ARG] [ARG]
```

The documentation states
> NOTE: When this option is used, preferences for the subparser which effect the parser behaviour (such as noIntersperse) are ignored.

IIUC this also disables `forwardOptions` which we rely on to forward flags to wrapped executables.